runtime/onert/core/include/compiler/Compiler.h

```diff
@@ -40,6 +40,10 @@ enum class State
 
 struct ManualSchedulerOptions
 {
+public:
+  void setBackendMap(const ir::Subgraphs &subgs, const std::string &str);
```

Comment:
I don't want to break field-only struct `ManualSchdulerOptions`. Do you have any reason to define here?

Reply:
I moved because it looks more readable. I also have question. Why do you want to keep `ManualSchedulerOptions` as field-only? It already has several c++ members, it is not a POD. It is already ordinary c++ class except that it has `public` access by default.

Reply:
Not special reason. In my memory, we decided to use struct as field-only, but now it is broken on many place. Ignore my comment.